openmetadata-ui/src/main/resources/ui/src/components/BotListV1/BotListV1.component.tsx

```diff
@@ -152,45 +172,88 @@ const BotListV1 = ({ showDeleted }: BotListV1Props) => {
     fetchBots(showDeleted);
   }, [showDeleted]);
 
-  return (
-    <>
-      <Row>
-        <Col span={24}>
-          <Table
-            columns={columns}
-            dataSource={botUsers}
-            loading={{
-              spinning: loading,
-              indicator: <Loader size="small" />,
-            }}
-            pagination={false}
-            size="small"
-          />
-        </Col>
-        <Col span={24}>
-          {paging.total > PAGE_SIZE && (
-            <NextPrevious
-              currentPage={currentPage}
-              pageSize={PAGE_SIZE}
-              paging={paging}
-              pagingHandler={handlePageChange}
-              totalCount={paging.total}
+  return handleErrorPlaceholder ? (
+    <ErrorPlaceHolder
+      buttons={
+        <div className="tw-text-lg tw-text-center">
+          <Button
+            ghost
+            title="Add Team"
+            type="primary"
+            onClick={handleAddBotClick}>
+            Add Bot
+          </Button>
+        </div>
+      }
+      doc={BOTS_DOCS}
+      heading="Bot"
+      type="ADD_DATA"
+    />
+  ) : (
```

Comment:
can we move this outside the return statement? in if the condition

Reply:
@ShaileshParmar11  we will take care of this point in code improvement post release.